addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -129,7 +130,11 @@ public MaxCubeBridgeHandler(Bridge br) {
     private boolean propertiesSet = false;
```

Comment:
Well, I have to say I'm in firm disagreement with Oracle here, but if it is code-convention, that'll take priority.